lib/authentication/view/edit_profile_page.dart

```diff
@@ -203,13 +213,64 @@ class _EditProfilePageState extends State<EditProfilePage> {
     );
   }
 
+  Future<void> getImage() async {
+    final Uint8List image = await StorageProvider.getImage();
+    setState(() {
+      if (image != null) {
+        this.image = image;
+      } else {
+        AppToast.show(S.of(context).errorImage);
+      }
+    });
+  }
+
+  Widget loadImage() {
```

Comment:
The name is misleading. This should be called `buildEditableAvatar` or something like that. It builds a widget, doesn't "load" something per se.